samples/speech_transcribe_async.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  filepath = 'path/to/audio.raw',
+  encoding = 'LINEAR16',
+  sampleRateHertz = 16000,
+  languageCode = 'en-US'
+) {
+  // [START speech_transcribe_async]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const filepath = 'path/to/audio.raw';
+  // const encoding = 'LINEAR16';
+  // const sampleRateHertz = 16000;
+  // const languageCode = 'en-US';
+
+  // Imports the Google Cloud Speech API client library.
+  const speech = require('@google-cloud/speech');
```

Comment:
The DLP canonical samples use an uppercase variable for the package and lowercase for the client.

Following these conventions, this sample would become:

```js
const Speech = require('@google-cloud/speech');

// ...

const speech = Speech.SpeechClient();
```

Inconsistency to be resolved.

Reply:
My opinion: I strongly prefer the approach this sample takes with `speech` and `client`. I prefer this > DLP

Reply:
Let me throw a wrench into the works and suggest this:
```js
const {SpeechClient} = require('@google-cloud/speech');
const speech = new SpeechClient();
```

Reply:
I prefer –

```
const client = new SpeechClient();
```

1. Consistency across the board
2. One less local variable (`speech`) in the scope of the sample to worry about having a naming collision with 😛 `#selfish`

```js
  // const language = 'en-US';
  // const inputText = 'Your text to analyze';

  // Imports the Google Cloud Language API client library.
  const {LanguageClient} = require('@google-cloud/language');

  // Import other required libraries.
  const fs = require('fs');
  const util = require('util');
  const readFile = util.promisify(fs.readFile);

  // Creates a client.
  const language = new LanguageClient.SpeechClient();

  /* certain doom */

```

Reply:
Cool!  So it looks like we agree on the import syntax.  I am however a little confused.  Are you advocating for `client` or `speech` as the name of the client object?

Reply:
`client` works, until you have a sample that needs two different clients from the same library, then they both can't be named `client` (I'm thinking of the Monitoring API).

Reply:
If we simply to the camelCase version of the Constructor, we get the best of both. No collisions, and the variable has the word `client` in it.

Reply:
I pushed such a change, PTAL

Reply:
> until you have a sample that needs two different clients from the same library

Ahhh... yes...

Reply:
> If we simply to the camelCase version of the Constructor, we get the best of both. No collisions, and the variable has the word client in it.

I don't follow, what does that look like? :)

Reply:
```js
const speechClient = new SpeechClient();
```

it's pushed to this PR now